kornia/augmentation/augmentation.py

```diff
@@ -1700,8 +1700,9 @@ def __init__(
         return_transform: bool = False,
         same_on_batch: bool = False,
         p: float = 0.5,
+        keepdim: bool = False
     ) -> None:
-        super().__init__(p=p, return_transform=return_transform, same_on_batch=same_on_batch, p_batch=1.0)
+        super().__init__(p=p, return_transform=return_transform, same_on_batch=same_on_batch, p_batch=1.0, keepdim=keepdim)
```

Comment:
`kornia/augmentation/augmentation.py:1705:121: E501 line too long (123 > 120 characters)` same for the others
https://results.pre-commit.ci/run/github/145693916/1638966171.MHMbINd0QVOjIaL5NqKkGA